hc/api/tests/test_badge.py

```diff
@@ -13,12 +13,15 @@ def setUp(self):
 
     def test_it_rejects_bad_signature(self):
         r = self.client.get("/badge/%s/12345678/foo.svg" % self.alice.username)
```

Comment:
avoid single letter variables. Aim for meaningful variable names for all your files.
This comment applies to all files affected by your changes.